### PR TITLE
fix: fixing items left after review of PR#310

### PR DESCRIPTION
**Pull request by theflockers**
This PR fix the outstanding items after merging the PR https://github.com/redhat-appstudio/release-service-catalog/pull/310

### CODE REVIEW

**Review by johnbieren — DISMISSED** [review]

File: pipelines/fbc-release/README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                           | Yes       | main                                                            |
 
 ### Changes in 1.6.0
-- modify the task `publish-index-image` to accept the new parameter `buildTimestamp` used
-  to append to the publishing image tag
+- add new parameter `buildTimestamp` when calling the `publish-index-image` task
+  * this new parameter is used for adding a timestamped tag to the index image
```
Comment (mmalina) [inline pipelines/fbc-release/README.md]: I didn't check when I suggested the change, but it turns out there is actually a slightly different convention of doing the indentation in this file. So this should be:
```
- add new parameter `buildTimestamp` when calling the `publish-index-image` task
  - this new parameter is used for adding a timestamped tag to the index image
```

File: tasks/publish-index-image/tests/test-publish-index-image.yaml [hunk 2]
```diff
@@ -58,14 +58,14 @@ spec:
...
               set -eux
 
-              internalRequests="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers \
-              | tr "\n" " ")"
+              internalRequests=$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers \
+              -o json | jq '.items')
 
-              read -r -a ir <<< "${internalRequests}"
-              for((i=0; i<${#ir[@]}; i++)); do
+              irIndex=$(jq '. | length' <<< "${internalRequests}")
```
Comment (mmalina) [inline tasks/publish-index-image/tests/test-publish-index-image.yaml:64]: Can we also check that there are exactly 2 of them and fail otherwise?

Comment (theflockers) [reply tasks/publish-index-image/tests/test-publish-index-image.yaml:64]: check added.

File: pipelines/fbc-release/README.md (hunk 1 above)
Comment (theflockers) [reply pipelines/fbc-release/README.md]: fixed

**Review by mmalina — APPROVED** [review]

**Review by happybhati — APPROVED** [review]

**Review by johnbieren — APPROVED** [review]

**Pull request merged**